### PR TITLE
Update UI for tab-request-approval nodejs sample

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

**Comment by Harikrishnan-MSFT** [comment]
**Build Pipeline- 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8234701336
image**

**Review by Prithvi-MSFT — APPROVED** [review]

File: samples/tab-request-approval/nodejs/views/UserNotification.ejs [hunk 1]
```diff
@@ -310,6 +345,7 @@
                 success: function (profile) {
                     appId = getAppId(profile);
                     sendNotification(userId, taskInfo, token, appId);
+                    console.log('success');
```
Comment (Pawank-MSFT) [inline samples/tab-request-approval/nodejs/views/UserNotification.ejs:348]: remove this.

File: samples/tab-request-approval/nodejs/views/UserNotification.ejs [hunk 2]
```diff
@@ -365,6 +401,7 @@
                 },
                 success: function (profile) {
                     saveRequestDetails(taskInfo);
+                    console.log('success');
```
Comment (Pawank-MSFT) [inline samples/tab-request-approval/nodejs/views/UserNotification.ejs:404]: remove this.

**Review by Pawank-MSFT — APPROVED** [review]
small comments, rest looks ok, approving!

File: samples/tab-request-approval/nodejs/views/UserNotification.ejs (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-request-approval/nodejs/views/UserNotification.ejs]: Fixed

File: samples/tab-request-approval/nodejs/views/UserNotification.ejs (hunk 2 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-request-approval/nodejs/views/UserNotification.ejs]: Fixed

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline-
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8370840189
image

**Pull request merged**